src/styles/form.scss

```diff
@@ -0,0 +1,51 @@
+.form {
+  width: 40%;
+
+  @media (max-width: $displayLaptop) {
+    margin-top: 32px;
+    width: auto;
+  }
+
+  &__input {
+    resize: vertical;
+    display: block;
+    width: 100%;
+    margin-bottom: 16px;
+    padding: 17px;
+    font-family: $defaultFontFamily;
+    background-color: $inputBgColor;
+    color: $regularTextColor;
+    border: 1px solid $lightColor;
+    border-radius: $border-radius;
+    outline: none;
+    &::placeholder {
+      font-family: $defaultFontFamily;
+      color: $inputTextColor;
+    }
+    &:focus {
+      box-shadow: 0 6px 23px $inputBgColor;
+    }
+    &--textarea {
+      height: 145px;
+    }
+  }
+
+  &__submit {
+    display: block;
+    width: 100%;
+    padding: 20px;
+    font-family: $defaultFontFamily;
```

Comment:
the basic font-family should be assigned to body or html and inherited by others (no matter manually or automatically)

Reply:
no! it dont works so. you can check.
https://prnt.sc/thk54m
I dont add anything if it is unecessary

Reply:
just write font: inherit

Reply:
ok